docs/data-sources/machine_disks.md

```diff
@@ -0,0 +1,99 @@
+---
+page_title: "talos_machine_disks Data Source - talos"
+subcategory: ""
+description: |-
+  Generate a machine configuration for a node type
+---
+
+# talos_machine_disks (Data Source)
+
+Generate a machine configuration for a node type
+
+-> **Note:** Since Talos natively supports `.machine.install.diskSelectors`, the `talos_machine_disks` data source maybe just used to query disk information that could be used elsewhere. It's recommended to use `machine.install.diskSelectors` in Talos machine configuration.
```

Comment:
```suggestion
-> **Note:** Since Talos natively supports `.machine.install.diskSelector`, the `talos_machine_disks` data source maybe just used to query disk information that could be used elsewhere. It's recommended to use `machine.install.diskSelector` in Talos machine configuration.
```

Reply:
good catch